scripts/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env sh
+
+esbuild_cmd() {
+  format=$1
+
+  outdir="dist/esm"
+  platform="browser"
+
+  if [ "$format" = "cjs" ]; then
+    outdir="dist/cjs"
+    platform="node"
+  fi
+
+  ./node_modules/.bin/esbuild \
+    --outdir=$outdir \
+    --platform=$platform \
+    --target=es2019 \
+    --sourcemap \
+    --bundle \
+    --format=esm \
+    --splitting \
+    --log-level=error \
+    --jsx-factory='React.createElement' \
+    --jsx-fragment='React.Fragment' \
+    --loader:.json=json \
+    --loader:.js=jsx \
+    --external:"react" \
+    --external:"assert" \
+    --external:"crypto" \
+    --external:"http" \
+    --external:"https" \
+    --external:"os" \
+    --external:"stream" \
+    --external:"url" \
+    --external:"util" \
+    src/index.js
+
+  if [ "$format" = "esm" ]; then
+    fix_esm_default "@walletconnect/web3-provider/dist/cjs/index.js"
+    fix_esm_default "walletlink/dist/index.js"
```

Comment:
I only saw notes below for WalletConnect, but WalletLink also has this problem?

Reply:
Yes, both have are using the `__esModule: true` format.

Reply:
Closing as we are using TSDX now.